packages/vscode-extension/src/webview/views/ManageDevicesView.tsx

```diff
@@ -59,8 +82,12 @@ function ManageDevicesView() {
     reload();
   }, []);
 
-  const androidDevices = devices.filter((device) => device.platform === DevicePlatform.Android);
-  const iosDevices = devices.filter((device) => device.platform === DevicePlatform.IOS);
+  const iosDevices = devices.filter(
+    ({ platform, name }) => platform === DevicePlatform.IOS && name.length > 0
```

Comment:
Why we need to filter out devices without name?

Reply:
Similar filter is used in `DeviceSelect`, which was added in Windows Integration PR. I believe it is related to emulator files left after device removal, which happens sometimes on Windows.